### PR TITLE
Add postgres support

### PR DESCRIPTION
**Pull request by luizvarela**
This PR tries to add PostgreSQL to the project. There're some things we need to note here, the current implementation will generate models to all database columns, for example with a Rails or Phoenix application it will generate a model file to `schema_migrations` and that's not necessary.

- [ ] How to add Postgres primary key support.
- [ ] Decide which columns from the database we will export.
- [ ] Add tests.

### CODE REVIEW

File: lib/database/postgresql.ex [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    def get_columns(db, table) do
+        {_, result} = Postgrex.query(db.connection, "select column_name, data_type from INFORMATION_SCHEMA.COLUMNS where table_name = '#{table.name}';", [])
+        result.rows
+        |> Enum.map(&to_column/1)
+    end
+
+    defp to_column(row) do
+        {_,name} = Enum.fetch(row,0)
+        type = Enum.fetch(row,1) |> get_type
+        #primary_key? = Enum.fetch(row,3) == "PRI" # Needs to know how to get the primary_key
```
Comment (jhartwell) [inline lib/database/postgresql.ex]: Please remove this comment as it is more of a TODO

File: lib/database/postgresql.ex [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        %Plsm.Database.Column {name: name, type: type, primary_key: false}
+    end
+
+    defp get_type(start_type) do
+        {_,type} = start_type
+        upcase = String.upcase type
+        cond do
+            String.starts_with?(upcase, "INT") == true -> :integer
```
Comment (jhartwell) [inline lib/database/postgresql.ex]: You are missing "integer" type, which may or may not be common. Take a look at https://www.techonthenet.com/postgresql/datatypes.php to see what other types may convert

**Comment by luizvarela** [comment]
@jhartwell Great, I'll  work on your comments

**Comment by luizvarela** [comment]
I updated the PR based on your comments, I will work on add support to the primary key. Soon I'll send new updates.

**Comment by jhartwell** [comment]
This has been merged. Thanks!

**Pull request merged**